fluidBuild.config.cjs

```diff
@@ -116,7 +114,12 @@ module.exports = {
 			script: true,
 		},
 		"depcruise": [],
-		"check:exports": ["api"],
+
+		// The package's local check:exports tasks may use the entrypoint from either CJS or ESM,
+		// therefore we need to build both before running these checks. These checks also depend on the entrypoints being
+		// gnerated, which is done in the api tasks (api-extractor:commonjs and api-extractor:esnext).
+		"check:exports": ["tsc", "build:esnext", "api"],
```

Comment:
I don't think we want this. We don't need to run both; we need to run the one that matters.
"api" currently depends on "api-extractor:commonjs" and "api-extractor:esnext" which should be transitively enough, right?
Where did this breakdown?
("api" also depends on "typetest:gen" and that looks wrong. There are no comments explaining why. My guess is because some packages build tests with production code. But those packages should have "typetest:gen" as required by "tsc" and/or "build:esnext". I see some removals of that and that seems incorrect.)

Reply:
The problem was in test-loader-utils, which lacked an api task. Fixed with this task config in the package:

```json
	"fluidBuild": {
		"tasks": {
			"check:exports:cjs:index": [
				"...",
				"tsc"
			],
			"check:exports:esm:index": [
				"...",
				"build:esnext"
			],
			"check:exports:bundle-release-tags": [
				"...",
				"build:esnext"
			]
		}
	},
```